src/ngraph/op/constant.hpp

```diff
@@ -126,6 +126,9 @@ namespace ngraph
                 set_output_type(0, m_element_type, m_shape);
             }
 
+            Shape get_shape_val() const;
```

Comment:
Please add descriptions here of what these are for.

Reply:
I'm not sure when we could get negative values. Do we know clamping them to 0 make sense? Should we throw an exception here?